snmp/tools/mib-transformer.py

```diff
@@ -0,0 +1,70 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import sys
+
+import yaml
+from pysnmp import hlapi
+from pysnmp.smi import view
+
+
+def main(yaml_file):
+    """
+    Do OID translation in a SNMP profile. This isn't a plain replacement, as it
+    doesn't preserve comments and indent, but it should automate most of the
+    work.
```

Comment:
This should go in https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta imo

Reply:
Is it worth adding snmp deps to ddev?

Reply:
Import in function

Reply:
and add note in docstring